Four Brains/Model/K.swift

```diff
@@ -9,24 +9,110 @@
+        static let BEAT_CARD_2B = BeatCard(label: "2B", noteSequence: [0,1,1,0])
+        static let BEAT_CARD_2C = BeatCard(label: "2C", noteSequence: [0,0,1,1])
+        static let BEAT_CARD_2D = BeatCard(label: "2D", noteSequence: [1,0,0,1])
+        static let BEAT_CARD_2E = BeatCard(label: "2E", noteSequence: [1,0,1,0])
+        static let BEAT_CARD_2F = BeatCard(label: "2F", noteSequence: [0,1,0,1])
+        static let BEAT_CARD_3A = BeatCard(label: "3A", noteSequence: [1,1,1,0])
+        static let BEAT_CARD_3B = BeatCard(label: "3B", noteSequence: [1,1,0,1])
+        static let BEAT_CARD_3C = BeatCard(label: "3C", noteSequence: [1,0,1,1])
+        static let BEAT_CARD_3D = BeatCard(label: "3D", noteSequence: [0,1,1,1])
+        static let BEAT_CARD_4 = BeatCard(label: "4", noteSequence: [1,1,1,1])
+        
+        var BEAT_CARD_ARRAY: [BeatCard]
+        
+        
+        init() {
+            BEAT_CARD_ARRAY = [K.BeatCardInstances.BEAT_CARD_0,
+                               K.BeatCardInstances.BEAT_CARD_1A,
+                               K.BeatCardInstances.BEAT_CARD_1B,
+                               K.BeatCardInstances.BEAT_CARD_1C,
+                               K.BeatCardInstances.BEAT_CARD_1D,
+                               K.BeatCardInstances.BEAT_CARD_2A,
+                               K.BeatCardInstances.BEAT_CARD_2B,
+                               K.BeatCardInstances.BEAT_CARD_2C,
+                               K.BeatCardInstances.BEAT_CARD_2D,
+                               K.BeatCardInstances.BEAT_CARD_2E,
+                               K.BeatCardInstances.BEAT_CARD_2F,
+                               K.BeatCardInstances.BEAT_CARD_3A,
+                               K.BeatCardInstances.BEAT_CARD_3B,
+                               K.BeatCardInstances.BEAT_CARD_3C,
+                               K.BeatCardInstances.BEAT_CARD_3D,
+                               K.BeatCardInstances.BEAT_CARD_4]
+        }
+        
+    }
+    
+    class BeatCardNoteStrings {
+        static let BEAT_CARD_0_NOTE_STRING = "- - - -"
+        static let BEAT_CARD_1A_NOTE_STRING = "X - - -"
+        static let BEAT_CARD_1B_NOTE_STRING = "- X - -"
+        static let BEAT_CARD_1C_NOTE_STRING = "- - X -"
+        static let BEAT_CARD_1D_NOTE_STRING = "- - - X"
+        static let BEAT_CARD_2A_NOTE_STRING = "X X - -"
+        static let BEAT_CARD_2B_NOTE_STRING = "- X X -"
+        static let BEAT_CARD_2C_NOTE_STRING = "- - X X"
+        static let BEAT_CARD_2D_NOTE_STRING = "X - - X"
+        static let BEAT_CARD_2E_NOTE_STRING = "X - X -"
+        static let BEAT_CARD_2F_NOTE_STRING = "- X - X"
+        static let BEAT_CARD_3A_NOTE_STRING = "X X X -"
+        static let BEAT_CARD_3B_NOTE_STRING = "X X - X"
+        static let BEAT_CARD_3C_NOTE_STRING = "X - X X"
+        static let BEAT_CARD_3D_NOTE_STRING = "- X X X"
+        static let BEAT_CARD_4_NOTE_STRING = "X X X X"
```

Comment:
Will these variables ever be used? Where the X is, is very reminiscent of where the one is in the note sequence array above. Could these strings just be generated via the above definitions?

Reply:
I will create a new branch after this merge and refactor this to generate the beat cards. Thanks!